### PR TITLE
[US266 Task285] Updated bar chart to have custom properties and used it to visualize adopted work

### PR DESCRIPTION
**Pull request by aarti-kalekar**
Updated bar chart to have custom properties like title, axis data keys, axis labels, and bar chart legend keys. Used it for adopted work and updated usage in burndown. For now uses test data.

image

### CODE REVIEW

**Review by vrupak — APPROVED** [review]
Looks good to me!

File: react-ui/src/graph/testData.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+// TODO - delete this file when done testing
+export const testData = [
```
Comment (shahih25) [inline react-ui/src/graph/testData.js:2]: Do we need this now?

Comment (aarti-kalekar) [reply react-ui/src/graph/testData.js:2]: Yes, until API integration is complete.

Comment (aarti-kalekar) [reply react-ui/src/graph/testData.js:2]: (Task 308)

**Review by shahih25 — APPROVED** [review]
lgtm

**Pull request merged**